src/algorithms/spectral/viterbi.cpp

```diff
@@ -41,42 +42,54 @@ void Viterbi::compute() {
   std::vector<std::vector<Real> >& lattProbs = _lattProbs.get();
   std::vector<std::vector<Real> >& lattPtrs = _lattPtrs.get();
```

Comment:
initialize to -1 whole matrix. Initializing to 0 is ambiguous because 0 means in this context pointer to 0-th state